### ISSUE TITLE
10$: Error on missing comma after object literal method

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#277]
The following code is missing a semicolon:
```javascript
let o = {
  a() {}  // missing ,
  b() {}
};
```
Make quick-lint-js report an error for this code.

Related: #912

**Comment by singalhimanshu** [comment]
> The following code is missing a semicolon:

Did you meant to say comma here instead of semicolon?

> Make quick-lint-js report an error for this code.

Also what error should be reported here? The current error reported by qljs is this - `missing comma between object literal entries [E025]`

**Comment by strager** [comment]
> > The following code is missing a semicolon:
> 
> Did you meant to say comma here instead of semicolon?

Yes, I meant comma.

> > Make quick-lint-js report an error for this code.
> 
> Also what error should be reported here? The current error reported by qljs is this - `missing comma between object literal entries [E025]`

Hmm, you're right. Maybe my example is incorrect. I'll have to look at the code I was editing when I made this report. I suspect the real issue is #191.

**Comment by strager** [comment]
As @singalhimanshu said, this is already implemented.
